### PR TITLE
BREAKING: refactor repo to use SkeletonUI

### PR DESCRIPTION
**Pull request by MDr164**
The rationale behind this is already explained in the commit message, this is basically a full project rework in terms of the UI framework used. With this we move from carbon-svelte to SkeletonUI.

### CODE REVIEW

**Comment by MDr164** [comment]
Looks like the commit lint fails to execute due to it trying to use our tsconfig as input. Might need explicit configuration or we should switch the action to a more robust one. Same with coverage, it should not factor in the pnp files but it also does not factor in the svelte files. Will need to update that too.

**Review by nd55 — APPROVED** [review]
Nice!

🚀 🚀 🚀

**Review by nd55 — APPROVED** [review]

**Review by nd55 — APPROVED** [review]

File: src/app.postcss
[the remark this answers is not in the document]
Comment (MDr164) [reply src/app.postcss:1]: That's what the scaffolder from Skeleton generated. I suppose there is only ever a single global app.postcss so there won't be too much benefit by shortening that.

File: src/app.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     %sveltekit.head%
   </head>
-  <body data-sveltekit-preload-data="tap">
-    <div>%sveltekit.body%</div>
+  <body data-sveltekit-preload-data="tap" , data-theme="skeleton">
```
[the remark this answers is not in the document]
Comment (MDr164) [reply src/app.html]: Done

**Comment by MDr164** [comment]
Prettier is fixed, as well as coverage for yarn pnp and the commit linter which now just uses the git command and regex vs a node implementation. Makes it also a lot faster.

**Review by nd55 — APPROVED** [review]

**Pull request merged**